Extension point for kubernetesObjectsProvider (#19969)
backstage/backstage: .changeset/nice-ghosts-build.md

---
'@backstage/plugin-kubernetes-node': minor
'@backstage/plugin-kubernetes-backend': patch
---

> A new Package is introduced to house the backend plugin's extension points for Kubernetes plugin, at the moment only the KubernetesObjectsProviderExtensionPoint is present. The Kubernetes-backed package was modified to use this new extension point

Reviewer: Started with a typo fix ('Kubernetes-backed') and then I got carried away with wordsmithing
```suggestion
A new plugin has been introduced to house the extension points for Kubernetes backend plugin; at the moment only the `KubernetesObjectsProviderExtensionPoint` is present. The `kubernetes-backend` plugin was modified to use this new extension point.
```
